draft-ietf-add-resolver-info.md

```diff
@@ -106,20 +106,17 @@ Encrypted DNS resolver:
    RR type and the QNAME of the domain name that is used to authenticate the
    DNS resolver (referred to as the Authentication Domain Name (ADN) in {{!RFC9463}}).
 
-   If the Special-Use Domain Name "resolver.arpa", defined in
-   {{!RFC9462}}, is used to discover an encrypted DNS resolver, the
-   client can retrieve the resolver information using the RESINFO RR
-   type and QNAME of "resolver.arpa".  If a resolver supports DDR
-   but does not support RESINFO, the client can receive a positive RESINFO
-   response from a legitimate upstream DNS resolver or an attacker. While DNSSEC can be considered as a candidate mechanism
-   to protect against the attack, it is important to note that DNSSEC protection is not possible with
-   "resolver.arpa" due to the lack of uniqueness. To prevent such an attack,
-   resolvers supporting DDR MUST convey the "sig" attribute as defined in {{key-val}}. DNS clients
-   using DDR for encrypted resolver discovery querying for a RESINFO RR MUST validate the signature
-   in the "sig" attribute for data origin authentication using the public key
-   in the certificate, including ensuring that the certificate contains
-   the IP address that DDR uses for validation as per {{Section 4.2 of RFC9462}}.
-   If the signature validation fails, the DNS client MUST reject the RESINFO RR.
+   Clients face a special circumstance when the Special-Use Domain Name "resolver.arpa" is used to discover the encrypted
+   DNS server using the Discovery Using Resolver IP Address mechanism defined in {{Section 4 of !RFC9462}}.
+   By using the DNS server's domain name from the DDR SVCB response to issue the RESINFO query,
+   a client accepts the risk that a resolver supports DDR
+   but does not support RESINFO. In this scenario, the server might pass the query upstream, and then the client can receive a positive RESINFO
+   response either from a legitimate upstream DNS resolver or an attacker. While DNSSEC can be considered as a candidate mechanism
+   to protect against the attack, it is important to note that the name was received over plain-text
+   DNS and that the RESINFO response can be both validly DNSSEC-signed and not signed by the name the original DDR redirection intended. 
+   To reduce the scope of such an attack, clients wishing to retrieve resolver information from resolvers discuvered using DDR's 
```

Comment:
```suggestion
   To reduce the scope of such an attack, clients wishing to retrieve resolver information from resolvers discovered when performing DDR
```

Reply:
Added this change manually since it conflicted with another suggested edit that outdated this